apps/bare-expo/package.json

```
@@ -83,10 +83,11 @@
   "devDependencies": {
     "@babel/core": "^7.20.0",
     "@types/react": "~19.1.10",
-    "babel-plugin-module-resolver": "^5.0.2",
     "babel-preset-expo": "~54.0.1",
     "expo-module-scripts": "^5.0.7",
-    "jest": "^29.3.1"
+    "glob": "^11.0.3",
+    "jest": "^29.3.1",
+    "pixelmatch": "^7.1.0"
```

```suggestion
    "jest": "^29.3.1"
```

do we need the pixelmatch in bare-expo? should it be inside the image-comparison package?